File: CRM/src/app/shared/eHospital_DongNai_A_NSTL/NS_L_DMCHITIEU.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_L_DMCHITIEU } from './NS_L_DMCHITIEU.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_L_DMCHITIEUService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'MaChiTieu', 'TenChiTieu', 'CongThuc', 'GiaTri', 'ThuTuTinhToan', 'ThuTuTrenBaoCaoLuong', 'HienThiHayKhong', 'ThuTuSapXep', 'LoaiChiTieu', 'ThuTuTrenPhieuLuong', 'TinhChat', 'ChuyenThangSau'];

    List: NS_L_DMCHITIEU[] | undefined;
    ListFilter: NS_L_DMCHITIEU[] | undefined;
    FormData!: NS_L_DMCHITIEU;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_L_DMCHITIEU";
    }
}

File: Service_eHospital_DongNai_A_NSTL/Implement/NS_L_DMCHITIEUService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_L_DMCHITIEUService : BaseService<NS_L_DMCHITIEU, INS_L_DMCHITIEURepository>
    , INS_L_DMCHITIEUService
    {
    private readonly INS_L_DMCHITIEURepository _NS_L_DMCHITIEURepository;
    public NS_L_DMCHITIEUService(INS_L_DMCHITIEURepository NS_L_DMCHITIEURepository) : base(NS_L_DMCHITIEURepository)
    {
    _NS_L_DMCHITIEURepository = NS_L_DMCHITIEURepository;
    }
    }
    }

